examples/ffva/src/control/packet_queue.h

```diff
@@ -0,0 +1,110 @@
+// Copyright 2024 XMOS LIMITED.
+// This Software is subject to the terms of the XCORE VocalFusion Licence.
+#pragma once
```

Comment:
You can clean up all the packet queue related stuff since its only needed when the servicer talks to an underlying resource via a shared memory msg queue.